ci/nosetests_only.sh

```diff
@@ -1,12 +1,16 @@
 #!/usr/bin/env bash
 
+EXIT_CODE=0
+
 echo
 echo "========================================="
 echo "nosetests -a '!requires_sudo' output"
 echo "========================================="
-nosetests -a '!requires_sudo' tests
+nosetests -a '!requires_sudo' tests || EXIT_CODE=$(($EXIT_CODE || $?))
 
 echo "========================================="
 echo "nosetests -a 'requires_sudo' output"
 echo "========================================="
-sudo env "PATH=$PATH" nosetests -a 'requires_sudo' tests
\ No newline at end of file
+sudo env "PATH=$PATH" nosetests -a 'requires_sudo' tests || EXIT_CODE=$(($EXIT_CODE || $?))
```

Comment:
Same thing as above. Note that we or the command's exit code with our variable so as not to overwrite any exit code we grabbed from the prior `nosetests` command.

Reply:
@kevinclark19a - we are ok, somehow i missed the initialization of EXIT_CODE on line 3.

could add set -u to the 2 scripts? thanks